guess-the-country/country-backend/country/api.py

```diff
@@ -114,23 +72,66 @@ def streetview():
                     (13.4681858, 52.4993089), (13.4683145, 52.4996224), (13.4687008, 52.4996485), (13.4687437, 52.4998314), (13.4686579, 52.5000926), (13.4688099, 52.5007736), (13.4691103, 52.501492), (13.4696038, 52.502341), (13.4698399, 52.502328), (13.4700759, 52.5030986), (13.4693571, 52.5031835)]
     }
 
-    tel_aviv = {
+tel_aviv = {
         "country": "Israel",
-        "city": "Tel-Aviv",
+        "city": "Tel Aviv",
```

Comment:
Ich würde hier zwischen der Bezeichnung, die du für die User ausgibst, und dem Identifier, den du intern nutzt, unterscheiden. Ich glaube, es ist am einfachsten, wenn das Backend nur `tel_aviv`, `west_jerusalem` usw. kennt und die Übersetzung in eine schöne Darstellung (die ja auch von der Anzeigesprache abhängt, also ob Englisch, Deutsch, Hebräisch...) im Frontend gemacht wird.

Das löst dann mittelfristig auch [dieses Problem](https://lgtm.com/projects/g/XAI-Demonstrator/xai-demonstrator/alerts/?mode=list).

Reply:
Danke für den Tipp, hab ich jetzt so umgesetzt . Die Länder und Städte werden in einer Array verwaltet mit dem zugehörigen internen Bezeichner der für das Matching sorgt. Jetzt kann man auch ganz leicht weitere Länder und Städte ergänzen. :)